bosh-stemcell/spec/os_image/ubuntu_bionic_spec.rb

```diff
@@ -360,7 +371,38 @@
 
   describe 'allowed user accounts' do
     describe file('/etc/passwd') do
-      its(:content) { should eql(<<HERE) }
+      if ENV.key?("UBUNTU_ADVANTAGE_TOKEN")
```

Comment:
Do we need the duplication here? I was not able to identify a diff for the `passwd`, `shadow` and `gshadow` files.

Reply:
if you set `UBUNTU_ADVANTAGE_TOKEN`, there is a diff (things are ordered differently).